### PR TITLE
Update free-programming-books-langs.md

### PR DESCRIPTION
**Pull request by ItachiUchiha456**
## What does this PR do?
Add resource(s) 

### How do we know it's really free?
        The creator uploaded it for every learner for absolutely free

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1135,6 +1135,7 @@ Books on general-purpose programming that don't focus on a specific language are
 
 * [Airbnb JavaScript Style Guide](https://airbnb.io/javascript/) - Airbnb (HTML)
 * [Basic JavaScript for the impatient programmer](http://www.2ality.com/2013/06/basic-javascript.html) - Axel Rauschmayer (HTML)
+* [Bible of JS](https://shorturl.at/ax056) - Sheryians Coding School (JavaScript)
```
Comment (sohansai) [inline books/free-programming-books-langs.md:1138]: we don't accept shortened links

Comment (ItachiUchiha456) [reply books/free-programming-books-langs.md:1138]: 'https://www.youtube.com/redirect?event=video_description&redir_token=QUFFLUhqbElBUUg3MlJqMTd0Sk90azliZ1hKUDBKdUs1UXxBQ3Jtc0tuSWZOcHhJeVRuamJob2Vfamc2UU5LN09KOXFlQjlVRjk0ejc1bmZIN1ozVEFXcGZYWDZGbjJUTlFRbjQ3dUM2NlJsNGF5SWw4WmhXbU01TXVGaEVkVFFNVEhSTnJTVUVBaFFiaTMydUxLSjY4WFFmQQ&q=https%3A%2F%2Fsheryians.com%2Fdownload%2Fbibleofjs_by_sheryians&v=Aw_SmcpEOdY'      So should i add this link? Is this valid?

Comment (sohansai) [reply books/free-programming-books-langs.md:1138]: It seems the link you provided redirects to a different site. We don't accept redirected links; we prefer original sources. Share the direct source for consideration.

Comment (ItachiUchiha456) [reply books/free-programming-books-langs.md]: Finally I got the original url. I've updated it check it out

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -1135,6 +1135,7 @@ Books on general-purpose programming that don't focus on a specific language are
 
 * [Airbnb JavaScript Style Guide](https://airbnb.io/javascript/) - Airbnb (HTML)
 * [Basic JavaScript for the impatient programmer](http://www.2ality.com/2013/06/basic-javascript.html) - Axel Rauschmayer (HTML)
+* [Bible of JS](https://sheryians.com/download/bibleofjs_by_sheryians) - Sheryians Coding School (JavaScript)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1138]:
```suggestion
* [Bible of JS](https://sheryians.com/download/bibleofjs_by_sheryians) - Harsh Sharma, Sheryians Coding School
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by ItachiUchiha456** [comment]
> Thanks, and welcome to free-programming-books!

Thank you, Sir

**Pull request merged**